### DISCUSSION TITLE
Taskbar on second screen only please !!

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#2697] [Ideas]
I use an Asus rog Zephyrus duo 16 which has two screens. Currently have taskbar on both screens but would be awesome to have it only on the second screen please. 

Thank you
<img width="1273" alt="Screenshot 2024-01-15 010849" src="https://github.com/valinet/ExplorerPatcher/assets/149981353/2fe6fbb3-6d13-49af-84d7-cedaee3bd6a2">

**Comment by pyrates999** [comment]
Look at [DisplayFusion](https://www.displayfusion.com/) if you want a more advanced taskbar.

**Reply by ghost** [reply]
Looking at that doesn't seem like i can have taskbar on second screen only.

**Reply by pyrates999** [reply]
Install it and find out.
